NHapi20/NHapi.Base/Model/Primitive/TSComponentOne.cs

```diff
@@ -279,7 +279,7 @@ public virtual DateTime GetAsDate()
 		{
 			try
 			{
-				string[] dateFormats = new string[] {LongDateTimeFormat, ShortDateTimeFormat, LongDateTimeFormatWithSecond, LongDateTimeFormatWithOffset };
+				string[] dateFormats = new string[] {LongDateTimeFormat, ShortDateTimeFormat, LongDateTimeFormatWithSecond, LongDateTimeFormatWithOffset, LongDateTimeFormatWithFactionOfSecond };

```

Comment:
Please correct spelling of "Faction" to "Fraction".

Reply:
Fixed